MobileFoodSchedules/MobileFoodSchedules/Models/MobileFoodPermit.cs

```diff
@@ -1,12 +1,4 @@
-﻿// <auto-generated />
```

Comment:
Good to remove commented-out code.  It's considered bad practice to push commented-out code to VCS, because you can always go back in history to see what used to be there.